extensions/wsc/src/main/java/org/mule/extension/ws/internal/WebServiceConsumer.java

```diff
@@ -17,6 +19,7 @@
  *
  * @since 4.0
  */
+@Export(classes = WscException.class)
```

Comment:
this shouldn't be necessary. If any of your operations have this on their throws signature it should be exported automatically